bouncer/static/scripts/redirect.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+function getSettings(document) {
+  return JSON.parse(
+    document.querySelector('script.js-bouncer-settings').textContent);
+}
+
+// The ID of the extension we want to talk to.
+var editorExtensionId = 'oldbkmekfdjiffgkconlamcngmkioffd';
+
+chrome.runtime.sendMessage(editorExtensionId, 'Hi?', function (response) {
```

Comment:
You might want to check that `chrome.runtime.sendMessage` exists before you try and call it!